example/example.h

```diff
@@ -0,0 +1,15 @@
+#ifndef GROUP_EXAMPLE_HEADER_GUARD
```

Comment:
The header guard is a bit too verbose. It is common to use the file's capitalized name and extension as the guard, as in:
```suggestion
#ifndef EXAMPLE_H
```
There's another possibility but it is more contentious: write
```suggestion
#pragma once
```
and remove all the header guard cruft. Works with almost all compilers but it's not in the C11 standard. I mention this here for your edification. I do not advocate that we go with `#pragma once`.